KE-Apache-Beam-Analysis/README.md

```diff
@@ -0,0 +1,75 @@
+# Data Analytics @ Edge
+
+![High level architecture](Images/High_level_Arch.png "High Level Architecture")
+
+The main aim of analytics engine is to get data from mqtt broker in stream format and apply rules on incoming data in real time and produce alert/action on mqtt broker. Getting data through pipeline and aplying analysis function is done by using apache beam.
+
+###  Apache beam:
+Apache Beam is an open source, unified model for defining both batch and streaming data-parallel processing pipelines. Using one of the open source Beam SDKs, wecan build a program that defines the pipeline.
+
+
+#### Why use Apache Beam for analytics:

```

Comment:
We need not have this section as we want to emphasize more on KubeEdge advantages than apache beam advantages